docs/release-notes/8.11.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[release-notes-header-8.11.0]]
 == 8.11
 
+[discrete]
+[[release-notes-8.11.4]]
+=== 8.11.3
+
+[discrete]
+[[bug-fixes-8.11.4]]
+==== Bug fixes
+* Stops the {esql} tab from rendering until you click on it in Timeline({pull}173484[#173484]).
```

Comment:
More of a nit, but would it make sense to group the ESQL ones together?

Reply:
Resolved in [90b86fe](https://github.com/elastic/security-docs/pull/4547/commits/90b86fe1b770f91facdb64ea9ac7816aaeda1bd1)